lib/scala/kg/src/main/scala/io/github/tetherlessworld/mcsapps/lib/kg/stores/neo4j/AbstractNeo4jKgStore.scala

```diff
@@ -3,17 +3,16 @@ package io.github.tetherlessworld.mcsapps.lib.kg.stores.neo4j
 import io.github.tetherlessworld.twxplore.lib.base.WithResource
 import org.neo4j.driver.{AuthTokens, GraphDatabase, Session, Transaction}
 import io.github.tetherlessworld.mcsapps.lib.kg.stores.Neo4jStoreConfiguration
-import io.github.tetherlessworld.mcsapps.lib.kg.models.search.KgSearchResultType
 import scala.collection.JavaConverters._
 
 abstract class AbstractNeo4jKgStore(protected val configuration: Neo4jStoreConfiguration) extends WithResource {
   protected val ListDelimChar = '|'
   protected val ListDelimString = ListDelimChar.toString
-  protected val NodeLabel = KgSearchResultType.Node.value
```

Comment:
Not sure why you disconnected these again, it seemed a reasonable thing to do to make the neo4j label be the KgSearchResultType.